### PR TITLE
[IT-3313] Setup infra for open data aws account

### PR DESCRIPTION
**Pull request by zaro0508**
Bootstrap and setup basic infrastructure for the opendata AWS account. These are the base resources that we put on every account to allow us to easily manage the account using infrastructure as code.

depends on https://github.com/Sage-Bionetworks/aws-infra/pull/401

### CODE REVIEW

**Comment by brucehoff** [comment]
> ukbiobank opendata AWS account.
Please note that while this account is for use with the Amazon Open Data program, it is not specifically for the ukbiobank data set.  ukbiobank is just one of the data sets to be hosted in this account.

**Comment by zaro0508** [comment]
> Please note that while this account is for use with the Amazon Open Data program, it is not specifically for the ukbiobank data set.  ukbiobank is just one of the data sets to be hosted in this account.

Do you have a specific suggestion @brucehoff ?  @xschildw created the account.

**Comment by xschildw** [comment]
My understanding from the document in the issue was that we needed one account per dataset.

**Comment by brucehoff** [comment]
> > Please note that while this account is for use with the Amazon Open Data program, it is not specifically for the ukbiobank data set.  ukbiobank is just one of the data sets to be hosted in this account.
> 
> Do you have a specific suggestion @brucehoff ? @xschildw created the account.

Yes, you will be seeing specific suggestions in the PR regarding naming things.  Aside from naming I don't think any changes need to be made.  And sorry for not making this clear at the outset.

**Comment by brucehoff** [comment]
> My understanding from the document in the issue was that we needed one account per dataset.

@xschildw You read the document correctly however I believe we have a different arrangement with Open Data.  I asked the question in Slack to double check with others working on this project.

File: sceptre/ukbiobank-opendata/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Overview
+A project to install, configure and manage the strides AWS account.
+
+## Purpose
+The purpose of this project is to build a self service provisioning
+system using the
+[AWS service catalog](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/introduction.html).
```
Comment (brucehoff) [inline sceptre/ukbiobank-opendata/README.md:7]: We don't need to deploy Service Catalog in this new account.

File: sceptre/ukbiobank-opendata/config/prod/jumpcloud-idp.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+template:
+  path: "jumpcloud-idp.yaml"
+stack_name: "jumpcloud-idp"
+stack_tags:
+  OwnerEmail: "it@sagebase.org"
+parameters:
+  AdminMetadata: !file_contents idp/jumpcloud/ukbiobank-opendata-admins.xml
```
Comment (brucehoff) [inline sceptre/ukbiobank-opendata/config/prod/jumpcloud-idp.yaml:7]: remove `ukbiobank` prefixes

File: sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-admins.xml [hunk 3]
```diff
@@ -0,0 +1 @@
...AQABowIwADANBgkqhkiG9w0BAQsFAAOCAgEAstQLq4E3tuDAHKZU5i53xgUZk9dSeA4g/iFwQtCgkC/kH3pnX2lbLC14c9x4uL0fUhQOFtxmAPdcDOx+/sWRWPNYAU1tVHrOD/bMvJsyyFnhPoWXi+ZkwJA8cJx448+1b1KQyBWRoTIzQzRDNsB0zul6z0Gd9S9/yTodCey3IgQvYZmA3j/jAm6GZKt4cPSyagXdvlW/sIFy29XUcX0A0Oycg5TNhg4AU9CxzapxkJ3NwLCPxCXlZsp9gNVDtqeU5+HdwT73hKruJtk+z70GXumOd8fEfVPVo3jQXLzko18kbgNW4jbKoyFAUm3/RTsaUgQoAGzV13EGYCcrB7SD7pKepwMZtvZu3Uk8rrgfEmxQcdeNsbhxQFxQILfg+RtXWSkKcE3ZpR3N5t+iRNXLDvzzDjuoPEbqJv1I+7s1GS5xMHLhKV09KDv13ZZX2yizhZhR4JERxSKRGgq3cheiFuvkeiBBUDmQ5pY7pPg9bI/IQ6U48UBeVm0VTQdWtwEFaHg3pw7kvObWegwXZWsYZ01G2LsallLXKGJXeEBouwzWBajHYrKRFrMboBs6c4NrkxEJ1hxpqWdZd/njEkwAqeiWwjU8THCE2jHfjCO+NtdHb8/CCVAs44dfYZwI0LUKN8n1LVDgDYUr+EmqmJ5N8SDXsdhAQlrCfAub1QvftAs=</ds:X509Certificate></ds:X509Data></ds:KeyInfo></md:KeyDescriptor><md:NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</md:NameIDFormat><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://sso.jumpcloud.com/saml2/strides-admin"/><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://sso.jumpcloud.com/saml2/strides-admin"/></md:IDPSSODescriptor></md:EntityDescriptor>
```
Comment (brucehoff) [inline sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-admins.xml:1]: rename to remove `ukbiobank` prefix.

File: sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-auditors.xml [hunk 4]
```diff
@@ -0,0 +1 @@
...AaMCMAAwDQYJKoZIhvcNAQELBQADggIBAJtTmVK/IOa2GspgH3uOUCUExd9JpaQVbrib3JzSGXnFOhytXf7UquOMW80kFKc96loGAKVat9s6kYuLMQ17DMJnEqZf3fjLKyK8LfsMFfj+aSGMDfuHiNTN0YgOwaZA3tOTHw5f69ZISvjc5B7MiZDnthLgWwuDUjxh+QX5esOyuUj06mJULnQvyRdzraqFKxoDHpjFzbU7ISDLO7A5n8PqydglMbov1upNUL3wFLPITcwU9rLnm8Yd7M668PTkIp/Hr4TATeiIiG/pHlwm8eG76GRLLTiu87cUJjJOgZuUYdPk0Embe5R/hM0QR5/Bz4xX4Ry0VbWqMlwYVzDZbmCQVt1fcfOixop5Ky7YZI/0R01Gc+v+KeED+wYHcKVIsM8ByI5xr2DZz2w6u7bbdaXYdk3vs/wiAbEicuFpNbinkXjXujmw8/R+Ss36+3jFKmS/s+cGHG3CheK2hDNLFVOFuJFjJbPUxw0bO3oYc1H1n+uJ8+f+uVClbMRwc3Yna44pKGOlQ2xhMDJ6qvW2NXy79EQ7xAqxNOZ8BxzcXeyFX15vpUDQnrnNYlx9jKrXKOr7F5ABmPQeef/8Vc8qw22HXqPknquMtYfq2m9S7+k1DP/ho6jsFYM78QGBLuMZMUAy0gCUgT3Yzeqw6EtUck8QZk/m0gUqRk3cduQP373D</ds:X509Certificate></ds:X509Data></ds:KeyInfo></md:KeyDescriptor><md:NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</md:NameIDFormat><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://sso.jumpcloud.com/saml2/strides-auditor"/><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://sso.jumpcloud.com/saml2/strides-auditor"/></md:IDPSSODescriptor></md:EntityDescriptor>
```
Comment (brucehoff) [inline sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-auditors.xml:1]: rename to remove ukbiobank prefix.

File: sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-viewers.xml [hunk 5]
```diff
@@ -0,0 +1 @@
...ABowIwADANBgkqhkiG9w0BAQsFAAOCAgEAEXstOZcOALOo/eaKUN6bwZaOpX8nZ7yeWREvGQLZuC1Irg/QlqbaYRUmCpAuVh1pswfcaIpnAdKi854DNvdcHZgp1IVGt0tkggwstT7FTd0o17wWcUDpTBar+tHfUCMftNKoQqidWSbo1WLeeMlC65pl+FTxJzFw5gmv/edt2zAaSwDKBEN+oxxp4UQNBMS18tnWRytd3nzVjkCuey0Rc1zdRxTVpwzmpYlW4DVyJrksyBX2fK1+A2HA5bpTOQLI9By6b1EYAwBZYbx8CA+b25ip6N/Ri/Jh4CYJ7+Xjof4nhTJR9mpDx6i82wbPXrhuRFe9HacaVQUC1MjarYh8dhU8zdJrCr9y9gWrZbtVk5ogYBNYlt36e6FCC3ir+uyad5tCMdNO63DZEzd64LlOujTgJ8GTZPZTXsrICccUBWf0Qskaaz0Z6zXPIKsAyDSBp9m5Al0iT3qTzzQtC5UuYgJPAaTu8uMUvC/Ut6iFIzrmQxiKn7JHR3M42NQ47iJQurVqPT5UXWGajUqTHk/tLUDhi/H9+NBznWdpkBhqTtOYkI3W0D9VwQVdJlOZNw8gx9NtStfx7kGCfrDZRNsmvu9tGAlNvrwdr66YaPC0QZsVZUreqpfJyUJp5j2CtHnWeJPggfptBToKraIRYAtT7+99YexLzjZ6wiFKjeVZiJg=</ds:X509Certificate></ds:X509Data></ds:KeyInfo></md:KeyDescriptor><md:NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</md:NameIDFormat><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://sso.jumpcloud.com/saml2/strides-viewer"/><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://sso.jumpcloud.com/saml2/strides-viewer"/></md:IDPSSODescriptor></md:EntityDescriptor>
```
Comment (brucehoff) [inline sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-viewers.xml:1]: rename to remove ukbiobank prefix.

File: sceptre/ukbiobank-opendata/templates/jumpcloud-idp.yaml [hunk 6]
```diff
@@ -0,0 +1,160 @@
+Description: Resources for Jumpcloud IDP
```
Comment (brucehoff) [inline sceptre/ukbiobank-opendata/templates/jumpcloud-idp.yaml:1]: rename to remove ukbiobank prefix.

**Review by brucehoff — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yaml [hunk 7]
```diff
@@ -240,3 +240,10 @@ jobs:
     with:
       role-to-assume: "arn:aws:iam::867686887310:role/sagebase-github-oidc-sage-bionetworks-it"
       working-dir: "sceptre/imagecentral"
+  sceptre-ukbiobank-opendata:
+    if: github.ref == 'refs/heads/master'
+    needs: [org-formation]
+    uses: "./.github/workflows/aws-deploy.yaml"
+    with:
+      role-to-assume: "arn:aws:iam::373143421983:role/github-oidc-sage-bionetworks-it"
+      working-dir: "sceptre/ukbiobank-opendata"
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply .github/workflows/main.yaml]: changed to `sceptre/aws-opendata` to keep it consistent with JC configuration.

File: .github/workflows/main.yaml [hunk 8]
```diff
@@ -240,3 +240,10 @@ jobs:
     with:
       role-to-assume: "arn:aws:iam::867686887310:role/sagebase-github-oidc-sage-bionetworks-it"
       working-dir: "sceptre/imagecentral"
+  sceptre-ukbiobank-opendata:
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply .github/workflows/main.yaml]: done

File: sceptre/ukbiobank-opendata/README.md (hunk 1 above)
Comment (zaro0508) [reply sceptre/ukbiobank-opendata/README.md]: removed

File: sceptre/ukbiobank-opendata/config/prod/jumpcloud-idp.yaml (hunk 2 above)
Comment (zaro0508) [reply sceptre/ukbiobank-opendata/config/prod/jumpcloud-idp.yaml]: doen

File: sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-viewers.xml (hunk 5 above)
Comment (zaro0508) [reply sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-viewers.xml]: done

File: sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-admins.xml (hunk 3 above)
Comment (zaro0508) [reply sceptre/ukbiobank-opendata/idp/jumpcloud/ukbiobank-opendata-admins.xml]: done

File: sceptre/ukbiobank-opendata/templates/jumpcloud-idp.yaml (hunk 6 above)
Comment (zaro0508) [reply sceptre/ukbiobank-opendata/templates/jumpcloud-idp.yaml]: done

File: sceptre/ukbiobank-opendata/README.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+# Overview
+A project to install, configure and manage the strides AWS account.
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply sceptre/ukbiobank-opendata/README.md]: done

**Review by brucehoff — APPROVED** [review]
Looks great!

**Review by xschildw — APPROVED** [review]
LGTM

**Pull request merged**